ai.finto.fi/static/js/annif.js

```diff
@@ -4,10 +4,14 @@ if (window.location.protocol.startsWith('http')) {
   // http or https - use APIs of current Annif and textract instances
   var annif_base_url = '/v1/';
   var textract_base_url = '/textract/';
-} else {
-  // local development case - use Finto AI dev API and textract running on localhost via port 8001
+} else if (window.location.protocol.startsWith('file')) {
+  // local development with browser - use Finto AI dev API
   var annif_base_url = 'https://ai.dev.finto.fi/v1/';
-  var textract_base_url = 'https://ai.dev.finto.fi/textract/'//'http://localhost:8001/textract/';
+  var textract_base_url = 'https://ai.dev.finto.fi/textract/'
+} else {
+  // local development with VS Code Live Server extension - use APIs of Annif on localhost via Live Server proxy (overcomes CORS error by /v1/detect-language)
```

Comment:
Do we want to have logic like this on the main branch?
(I'm fine either way, just asking whether this is intentional)

Reply:
Yes, lets have it on main branch. Using Annif on localhost allows to test on different Annif versions (with the http-backend proxy to ai.dev.finto.fi).

I've started to use the Live Server, because it autoreloads the page on code changes, and also allows to use the /v1/detect-language.